src/Rule/AbstractCompare.php

```diff
@@ -259,4 +277,21 @@ public function getHandler(): string
     {
         return CompareHandler::class;
     }
+
+    /**
+     * Formats list of strings as a single string where items separated by the comma and each item is wrapped with
+     * double quotes.
+     *
+     * For example, for `['item1', 'item2']` list, the output will be `"item1", "item2"`.
+     *
+     * @param string[] $items Initial list of strings to format.
+     *
+     * @return string Resulting formatted string.
+     */
+    private function getQuotedList(array $items): string
+    {
+        $wrapInQuotesCallable = static fn (string $item): string => '"' . $item . '"';
+
+        return implode(', ', array_map($wrapInQuotesCallable, $items));
```

Comment:
```suggestion
        return '"'. implode('", "', array_map($wrapInQuotesCallable, $items)) . '"';
```

Reply:
Should be:

```php
return '"'. implode('", "', $items) . '"';
```

But I got the idea and applied it.